client/tracing/src/block/mod.rs

```diff
@@ -229,7 +229,7 @@ where
 			.ok_or_else(|| Error::MissingBlockComponent("Extrinsics not found".to_string()))?;
 		tracing::debug!(target: "state_tracing", "Found {} extrinsics", extrinsics.len());
 		let parent_hash = *header.parent_hash();
-		let parent_id = BlockId::Hash(parent_hash);
+		let parent_id: BlockId<Block> = BlockId::Hash(parent_hash);
```

Comment:
nit:

you can remove this and just use `block_id_as_string(BlockId::<Block>::Hash(parent_hash))` in L301